### ISSUE TITLE
Macros like $(ProjectDir) are not working

### ISSUE THREAD

**Issue by dhilgarth** [clariuslabs/TransformOnBuild#30]
I have this in my template:
```
<#@ assembly name="$(ProjectDir)ContextGenerator.Entities.EF.dll" #>
```

It works when I save the T4 template in VS, but it is not working with TransformOnBuild. It says that it can't find the file.

**Comment by lukemcgregor** [comment]
+1 on this

**Comment by erictrade** [comment]
+1 on this =)

**Comment by olograph** [comment]
+1 on this

**Comment by mnaoumov** [comment]
@dhilgarth  @lukemcgregor  @erictrade @olograph Did you try to use my fork https://www.nuget.org/packages/Clarius.TransformOnBuild-unofficial/ ? It addresses this issue and many minor others

**Comment by erictrade** [comment]
hi @mnaoumov 
actually, after my comment here I found your fork and I'm using it until now. It works perfectly! Thanks for sharing!

**Comment by mnaoumov** [comment]
Fixed in #16
